doc/reference/security.rst

```diff
@@ -93,8 +91,233 @@ access to the field:
 
 .. todo::
 
-    field access groups apply to administrator in fields_get but not in
+    field access groups apply to the Superuser in fields_get but not in
     read/write...
 
-.. _foo: http://google.com
-.. _time module: https://docs.python.org/2/library/time.html
+.. _time module: https://docs.python.org/3/library/time.html
+
+
+.. _reference/security/pitfalls:
+
+Security Pitfalls
+=================
+
+As a developer, it is important to understand the security mechanisms and avoid
+common mistakes leading to unsecure code.
+
+Unsafe Public Methods
+---------------------
+
+Any public method can be execute via a :ref:`RPC call
```

Comment:
```suggestion
Any public method can be executed via a :ref:`RPC call
```

Reply:
Also maybe mention that there are no ACL checks for public methods, ACLs are only checked when specifically performing CRUD operations? So they're callable with essentially no access rights, possibly none at all using the webclient's RPC thing?